packages/lodestar-beacon-state-transition/package.json

```diff
@@ -19,7 +19,7 @@
     "lib/**/*.js.map"
   ],
   "scripts": {
-    "build": "yarn build:lib && yarn check-types && yarn build:types",
+    "build": "yarn build:lib && yarn build:types",
```

Comment:
just a very small note: if we remove `check-types` we'll not compile unit tests in `build`

Reply:
ignore the above, it's fine to remove check-types